src/Hosting/Hosting/src/Internal/HostingMetrics.cs

```diff
@@ -66,10 +67,12 @@ public void RequestEnd(string protocol, string scheme, string method, string? ro
 
             // Add information gathered during request.
             tags.Add("http.response.status_code", GetBoxedStatusCode(statusCode));
-            if (route != null)
+            var httpRoute = route;
+            if (string.IsNullOrEmpty(httpRoute))
```

Comment:
Should a `null` route be converted or just the empty string?

Reply:
I was wondering the same thing when re-reading my code. I was about to ask